types/types.go

```diff
@@ -34,6 +34,9 @@ type ImageDestination interface {
 	// Note: Calling PutBlob() and other methods may have ordering dependencies WRT other methods of this type. FIXME: Figure out and document.
 	PutBlob(digest string, stream io.Reader) error
 	PutSignatures(signatures [][]byte) error
+	// SupportedImageDestinationMIMEType tells which mime types the destination support when uploading
+	// If an empty slice or nil it's returned, then any mime type can be tried to upload
+	SupportedImageDestinationMIMEType() []string
```

Comment:
> Return a slice, so this should be plural.

https://github.com/containers/image/pull/26#discussion_r68942687

-

`SupportedMIMETypes()` is ok for me